auth/provider_config.go

```diff
@@ -826,20 +901,30 @@ func (c *baseClient) makeRequest(
 }
 
 type oidcProviderConfigDAO struct {
-	Name        string `json:"name"`
-	ClientID    string `json:"clientId"`
-	Issuer      string `json:"issuer"`
-	DisplayName string `json:"displayName"`
-	Enabled     bool   `json:"enabled"`
+	Name            string                   `json:"name"`
+	ClientID        string                   `json:"clientId"`
+	Issuer          string                   `json:"issuer"`
+	DisplayName     string                   `json:"displayName"`
+	Enabled         bool                     `json:"enabled"`
+	ClientSecret    string                   `json:"clientSecret"`
+	ResponseEnabled oidcProviderResponseType `json:"responseType"`
```

Comment:
Nit: I believe this can stay as `ResponseType` since it is not a boolean type. Not a big deal though as it is internal implementation.

Reply:
Oops, youre right.  Fixed